features/org.wso2.carbon.siddhi.extensions.installer.core.feature/src/main/resources/extensionsInstaller/extensionDependencies.json

```diff
@@ -319,6 +319,181 @@
     ]
   },
 
+  "cdc-oracle": {
+    "extension": {
+      "name": "cdc-oracle",
+      "displayName": "Change Data Capture - Oracle",
+      "version": "2.0.3"
+    },
+    "dependencies": [
+      {
+        "name": "siddhi-io-cdc",
+        "version": "2.0.3",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/io/siddhi/extension/io/cdc/siddhi-io-cdc/2.0.3/siddhi-io-cdc-2.0.3.jar"
+        },
+        "usages": [
+          {
+            "type": "BUNDLE",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "siddhi([_-])io([_-])cdc([_-])2.0.3.jar"
+      },
+      {
+        "name": "ojdbc8",
+        "version": "12.2.0.1",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://maven.xwiki.org/externals/com/oracle/jdbc/ojdbc8/12.2.0.1/ojdbc8-12.2.0.1.jar"
+        },
+        "usages": [
+          {
+            "type": "JAR",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "ojdbc8([_-])12.2.0.1(.+).jar"
+      },
+      {
+        "name": "xstreams",
+        "version": "12.2.0.1",
+        "download": {
+          "autoDownloadable": false,
+          "instructions": "https://www.oracle.com/database/technologies/instant-client/linux-x86-64-downloads.html download relevant instant client zip from above link and unzip to locate xstreams.jar. Convert it to bundle using jartobundle.sh and add it in <SI_HOME>/libs"
+        },
+        "usages": [
+          {
+            "type": "JAR",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "ojdbc8([_-])12.2.0.1(.+).jar"
+      }
+    ]
+  },
+
+  "cdc-postgresql": {
+    "extension": {
+      "name": "cdc-postgresql",
+      "displayName": "Change Data Capture - PostgreSQL",
+      "version": "2.0.3"
+    },
+    "dependencies": [
+      {
+        "name": "siddhi-io-cdc",
+        "version": "2.0.3",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/io/siddhi/extension/io/cdc/siddhi-io-cdc/2.0.3/siddhi-io-cdc-2.0.3.jar"
+        },
+        "usages": [
+          {
+            "type": "BUNDLE",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "siddhi([_-])io([_-])cdc([_-])2.0.3.jar"
+      },
+      {
+        "name": "postgresql",
+        "version": "42.2.10",
+        "download": {
+          "autoDownloadable": true,
+          "url": "https://repo1.maven.org/maven2/org/postgresql/postgresql/42.2.10/postgresql-42.2.10.jar"
+        },
+        "usages": [
+          {
+            "type": "BUNDLE",
+            "usedBy": "RUNTIME"
+          }
+        ],
+        "lookupRegex": "postgresql([_-])42.2.10(.+).jar"
```

Comment:
Won't there be `(.*)` instead of `(.+)` at the end, similar to the other ones we have changed above?